include/diaspora.php

```diff
@@ -751,9 +766,11 @@ private static function contact_by_handle($uid, $handle) {
 			intval($uid),
 			dbesc($nurl_sql)
 		);
-		if($r)
+		if(dbm::is_result($r)) {
```

Comment:
Standards: can you please add a space after the `if` operator? Sorry I didn't catch that on my first pass.